<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>795</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(133, 92, 119);</string>
  </property>
  <widget class="QWidget" name="shortBreakWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>600</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(125, 193, 83);
color:rgb(255,255,255);; </string>
   </property>
   <widget class="QLabel" name="timeLabel">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>90</y>
      <width>311</width>
      <height>201</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Chalkboard</family>
      <pointsize>80</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(195, 141, 179, 89);
border-radius:30px;</string>
    </property>
    <property name="text">
     <string>5:00</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>320</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Chalkboard</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(0, 0, 0);
border-radius:10px;
color:rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="skipButton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>420</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Chalkboard</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(0, 0, 0);
border-radius:10px;
color:rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Skip</string>
    </property>
   </widget>
   <widget class="QPushButton" name="goToMainMenuButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Chalkboard</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(0, 0, 0);
border-radius:10px;
color:rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Main Menu</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
